SampleApplications/Samples/NetCoreComplexClient/Program.cs

```diff
@@ -94,7 +95,8 @@ public static int Main(string[] args)
                 { "p|password=", "Password to access server.", (string n) => pw = n},
                 { "v|verbose", "Verbose output.", v => verbose = v != null},
                 { "j|json", "Print custom nodes as Json.", j => json = j != null},
-                { "r|reversible", "Use Json reversible encoding.", r => jsonReversible = r != null},
+                { "r|jsonreversible", "Use Json reversible encoding.", r => jsonReversible = r != null},
+                { "rc|reverseconnect=", "Connect using the reverse connection.", (string uri) => reverseConnectUri = new Uri(uri)},
```

Comment:
>new Uri(uri) [](start = 117, length = 12)

[Minor Bug] Potential Crash, it would be better to show the help if url is missformed